### PR TITLE
Data Sample update with fr-CA translations

### PR DESCRIPTION
**Pull request by KitioFofack**
This PR add few translations in french to the dataset.
These are useful to test the i18n on Capability Map

### CODE REVIEW

File: sample-data.n3 [hunk 1]
```diff
@@ -665,99 +713,119 @@
...
         a                       vivo:EditorRole ;
         rdfs:label              "Editor in Chief"@en-US ;
+        rdfs:label              "Editeur en Chef"@fr-CA ;
         obo:RO_0000052          <http://vivo.mydomain.edu/individual/n1736> ;
         vivo:dateTimeInterval   <http://vivo.mydomain.edu/individual/n2585> ;
         vivo:roleContributesTo  <http://vivo.mydomain.edu/individual/n6748> .
 
 <http://vivo.mydomain.edu/individual/n6561>
         a                    skos:Concept ;
         rdfs:label           "Electracy"@en-US ;
```
Comment (hauschke) [inline sample-data.n3:791]: The english label for http://vivo.mydomain.edu/individual/n6561 "Electracy" (https://en.wikipedia.org/wiki/Electracy), which shouldn't be translated to "Électricité".

File: sample-data.n3 [hunk 2]
```diff
@@ -897,50 +979,57 @@
...
         rdfs:label       "Martinez, Claudia"@en-US ;
+        rdfs:label       "Martinez, Claudia"@fr-CA ;
         obo:ARG_2000028  <http://vivo.mydomain.edu/individual/n4712> ;
         obo:RO_0000053   <http://vivo.mydomain.edu/individual/n2816> , <http://vivo.mydomain.edu/individual/n1515> ;
         vivo:relatedBy   <http://vivo.mydomain.edu/individual/n2543> , <http://vivo.mydomain.edu/individual/n6053> , <http://vivo.mydomain.edu/individual/n5058> .
 
 <http://vivo.mydomain.edu/individual/n6748>
         a                         bibo:Journal ;
         rdfs:label                "Journal of Political Rhetoric"@en-US ;
+        rdfs:label                "Journal de Politique Rhétorique"@fr-CA ;
```
Comment (hauschke) [inline sample-data.n3:1006]: It's disputable if journal titles should be translated.

**Review by hauschke — COMMENTED** [review]
Please consider the comments. I'd like to see an opinion by @mconlon17 on this, too.

File: sample-data.n3 (hunk 2 above)
Comment (KitioFofack) [reply sample-data.n3:1006]: That's a good point @hauschke ! 
From a technical perspective however, it does not make any difference if the string tagged as french is actually written in english or another language.

File: sample-data.n3 (hunk 1 above)
Comment (KitioFofack) [reply sample-data.n3:791]: Actually I was unable to translate that word in french that's why I wrote what google translate gave me. :)

Comment (hauschke) [reply sample-data.n3:791]: Maybe it's better to keep Electracy in French, too. Else I'm fine.

**Comment by chenejac** [comment]
Meanwhile sample-data repository has been restructured and internationalized. There is an i18n directory with sample-data for specific language including fr_CA. Therefore, this pull request is outdated and should be closed.

**Pull request closed without merging**